packages/frontend/src/ExploreGensApp.vue

```diff
@@ -298,6 +303,9 @@ export default {
     ]);
     this.ready = true;
   },
+  mounted() {
+    utils.removeClass("workaround-ex", "material-icons");
```

Comment:
please consider to add a 'TODO' comment to remove that stuff when root cause will solved